config/initializers/friendly_id.rb

```diff
@@ -0,0 +1,92 @@
+# FriendlyId Global Configuration
+#
+# Use this to set up shared configuration options for your entire application.
+# Any of the configuration options shown here can also be applied to single
+# models by passing arguments to the `friendly_id` class method or defining
+# methods in your model.
+#
+# To learn more, check out the guide:
+#
+# http://norman.github.io/friendly_id/file.Guide.html
+
+FriendlyId.defaults do |config|
+  # ## Reserved Words
+  #
+  # Some words could conflict with Rails's routes when used as slugs, or are
+  # undesirable to allow as slugs. Edit this list as needed for your app.
+  config.use :reserved
+
+  config.reserved_words = %w(new edit index session login logout users admin
```

Comment:
I'm not sure of the significance of this, but that config does work when I modify the routes like so:
```
get 'repositories/index'
resources :repositories, only: [:show]
```
in which case we actually get an extra route:
```
repositories_index GET    /repositories/index(.:format) repositories#index
```